packages/repository/README.md

```diff
@@ -10,8 +10,8 @@ the legacy `loopback-datasource-juggler` and connector modules from LoopBack
 3.x.
 
 This module provides data access facilities to various databases and services.
-It contains the constructs for modeling and accessing data. Repositories can be
-used alone or as part of a `Controller` implementation.
+It contains the constructs for modeling and accessing data. We recommend using
+Repositories with implementation of `Controllers`.
```

Comment:
implementation of `Controllers` but then can be used alone.

Reply:
I'm a bit confused. Are you saying that I should reword the sentence so that it doesn't imply controllers have to be used in conjunction with repositories?

Reply:
Yep. Adding a `but can be used alone` so it's clear our recommendation isn't the only way forward.